### PR TITLE
LPS-175206 Add Autom. Test FrontendDataSetAdmin#CanReturnNoMatchedResults

### PR DESCRIPTION
**Pull request by eltonccdantas**
Test plan

Given access Datasets admin page
When go to add Datasets
And When search for a phrase that doesn't match available headless providers
Then suggested list contains 0 results

[Poshi Report](https://drive.google.com/file/d/1GVapW3s2Fp9rDyjiL0N5iWNzG560yBu7/view?usp=share_link)

[Jira Ticket ](https://issues.liferay.com/browse/)

### CODE REVIEW

**Comment by eltonccdantas** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetAdmin.testcase [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetAdmin.testcase:20]:
```suggestion
		if (${testPortalInstance} == "true") {
```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetAdmin.testcase [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+		}
+
+		task ("And When search for a phrase that doesn't match available headless providers") {
+			FrontendDataSetAdmin.searchProvider(key_provider = "apple");
+		}
+
+		task ("Then suggested list contains 0 results") {
+			AssertElementNotPresent(
+				key_tagName = "${key_provider}",
+				locator1 = "AssetCategorization#TAG_AUTOCOMPLETE_SPECIFIC");
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetAdmin.testcase:73]:
```suggestion
				key_provider = "apple",
				locator1 = "AssetCategorization#TAG_AUTOCOMPLETE_SPECIFIC");
```

Comment (mateusralv) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetAdmin.testcase:73]: the variable `key_provider` is not global

**Review by mateusralv — CHANGES_REQUESTED** [review]
I think with these changes fix the sf 🤞🏽

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetAdmin.testcase (hunk 2 above)
Comment (eltonccdantas) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetAdmin.testcase:73]: It doesn't work locally... you can check the log [here](https://gist.github.com/eltonccdantas/70bca54adb967fed6b18e058bed09876)

Comment (eltonccdantas) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetAdmin.testcase:73]: I think it's something related to the Poshi Update, let's try it tomorrow

**Pull request closed without merging**